src/SCRIPTS/BF/ui.lua

```diff
@@ -313,9 +313,9 @@ function run_ui(event)
     elseif currentState == pageStatus.displayMenu then
         if event == userEvent.release.exit then
             currentState = pageStatus.display
-        elseif event == userEvent.release.plus or event == userEvent.dial.left then
+        elseif event == userEvent.release.plus or event == userEvent.dial.right then
```

Comment:
These changes will cause the rotary wheel to work backwards for other TX with a rotary wheel. If the NV14's rotary wheel is mounted backwards then this has to be worked around in a different way (and should probably be fixed in OpenTX, and not in these lua scripts).

Reply:
Changes reverted as requested.